src/mapping.ts

```diff
@@ -31,49 +31,59 @@ export function handleTransfer(event: Transfer): void {
   }
 
   let data = ipfs.cat(contentPath)
-  if (!data) return
-
-  let jsonResult = json.try_fromBytes(data!)
-  if (jsonResult.isError) return
-
-  let value = jsonResult.value.toObject()
-  if (data != null) {
-    let name = value.get('name')
-    if (name != null && name.kind == JSONValueKind.STRING) {
-      collectible.name = name.toString()
-    } else {
-      return
+  if (data) {
+    let jsonResult = json.try_fromBytes(data!)
+    if (jsonResult.isError) return
+  
+    let value = jsonResult.value.toObject()
+    if (data != null) {
+      let name = value.get('name')
+      if (name != null && name.kind == JSONValueKind.STRING) {
+        collectible.name = name.toString()
+      } else {
+        return
+      }
+  
+      let description = value.get('description')
+      if (description != null && description.kind == JSONValueKind.STRING) {
+        collectible.description = description.toString()
+      } else {
+        return
+      }
+  
+      let image = value.get('image')
+      if (image != null && image.kind == JSONValueKind.STRING) {
+        let imageStr = image.toString()
+        if (imageStr.includes(IPFS_SCHEME)) {
+          imageStr = getIpfsURL(imageStr)
+        }
+        collectible.imageURL = imageStr
+      } else {
+        return
+      }
     }
-
-    let description = value.get('description')
-    if (description != null && description.kind == JSONValueKind.STRING) {
-      collectible.description = description.toString()
-    } else {
-      return
+  
+    let name = erc721Token.try_name()
+    if (!name.reverted) {
+      collectible.collectionName = name.value
     }
-
-    let image = value.get('image')
-    if (image != null && image.kind == JSONValueKind.STRING) {
-      let imageStr = image.toString()
-      if (imageStr.includes(IPFS_SCHEME)) {
-        imageStr = getIpfsURL(imageStr)
-      }
-      collectible.imageURL = imageStr
+  
+    let symbol = erc721Token.try_symbol()
+    if (!symbol.reverted) {
+      collectible.collectionSymbol = symbol.value
+    }
+  } else {
+    if (event.address.toHexString() == '0x3f82395600296eef2e7022673bce036d3b040cf6') {
```

Comment:
I prefer not to have Bitzza hardcoded values in our code
Instead of this block can we do:
```
      collectible.name = 'some default value'
      collectible.description = 'some default value'
      collectible.collectionName = 'some default value'
      collectible.collectionSymbol = 'some default value'
      collectible.imageURL = 'some default value'
```

Reply:
@mul53 @LiorAgnin

Reply:
It'll break Freedom's wallet in production. They first need to make some changes in their client and publish it to prod, before we can apply those changes.